homeassistant/components/remote/__init__.py

```diff
@@ -110,3 +125,15 @@ def async_send_command(self, command, **kwargs):
         """
         return self.hass.async_add_job(ft.partial(
             self.send_command, command, **kwargs))
+
+    def learn_command(self, **kwargs):
+        """Learn a command from a device."""
+        raise NotImplementedError()
+
+    def async_learn_command(self, **kwargs):
+        """Learn a command from a device.
+
+        This method must be run in the event loop and returns a coroutine.
```

Comment:
Update the docstring. We shouldn't return a coroutine. We can remove the last line.

Reply:
Done.